homeassistant/components/ovo_energy/sensor.py

```diff
@@ -80,15 +80,15 @@ class OVOEnergySensorEntityDescription(SensorEntityDescription):
         key="last_gas_reading",
         translation_key="last_gas_reading",
         device_class=SensorDeviceClass.ENERGY,
-        state_class=SensorStateClass.TOTAL_INCREASING,
+        state_class=SensorStateClass.TOTAL,
```

Comment:
I don't think this one should be changed.
```suggestion
        state_class=SensorStateClass.TOTAL_INCREASING,
```